test/runtests.py

```diff
@@ -125,11 +126,16 @@
 if sys.platform != 'win32':
     not_tags.add('exclude_xplat')
     not_tags.add('Intl')
+    not_tags.add('require_simd')
+    not_tags.add('require_asmjs')
```

Comment:
This is all related to the move from excluding anything with the argument -simdjs to actually tagging the tests appropriately. As this changeset doesn't include the changes in #2716, it doesn't have working asmjs/wasm on xplat. As I expect that #2716 will get into master before this does, I'll rebase on top of that and remove the require_asmjs and require_wasm flags as appropriate from this changeset before checking in.